cvat-ui/src/reducers/notifications-reducer.ts

```diff
@@ -803,6 +803,36 @@ export default function (state = defaultState, action: AnyAction): Notifications
                 },
             };
         }
+        case ModelsActionTypes.CREATE_MODEL_FAILED: {
+            return {
+                ...state,
+                errors: {
+                    ...state.errors,
+                    models: {
+                        ...state.errors.models,
+                        canceling: {
+                            message: 'Could not create model',
+                            reason: action.payload.error.toString(),
+                        },
+                    },
+                },
+            };
+        }
+        case ModelsActionTypes.DELETE_MODEL_FAILED: {
+            return {
+                ...state,
+                errors: {
+                    ...state.errors,
+                    models: {
+                        ...state.errors.models,
+                        canceling: {
+                            message: 'Could not delete model',
```

Comment:
Let's pass id or name here to be more specific (in case when user deletes several models)